src/pyenphase/updaters/meters.py

```diff
@@ -14,11 +20,34 @@ class EnvoyMetersUpdater(EnvoyUpdater):
 
     end_point = ENDPOINT_URL_METERS
     data_end_point = ENDPOINT_URL_METERS_READINGS
+    production_meter_eid = None
+    consumption_meter_eid = None
+    production_meter_type: EnvoyMeterType | None = None
+    consumption_meter_type: EnvoyMeterType | None = None
+    phase_mode: EnvoyPhaseMode | None = None
+    phase_count = 1
+    ct_Meters_count = 0
+
+    def _set_my_common_properties(self) -> None:
```

Comment:
```suggestion
    def _set_common_properties(self) -> None:
```

my is already implied since its set on the object